c2corg_ui/templates/xreport/edit.html

```diff
@@ -208,7 +208,7 @@ <h2 class="heading show-phone" translate>xreport general</h2>
               <label translate>nb_participants</label>
               <div class="input-group">
                 <input class="form-control" type="number" min="0" max="1000" minlength="1" name="nb_participants" ng-model="xreport.nb_participants"/>
-                <span class="input-group-addon">persons</span>
+                <span class="input-group-addon" x-translate>persons</span>
```

Comment:
You should use ``translate`` instead of ``x-translate`` here, because you want ``persons`` to be extracted by the translations extractor tool. Ditto for the other change.

``translate`` and ``x-translate`` are equivalent (the latter being an alias of the first directive), except that ``x-translate`` is not recognized by the extractor.